inginious/frontend/templates/course.html

```diff
@@ -11,6 +11,7 @@
 $# Left column content
 $def ColumnF():
     $:template_helper.call('course_menu', course=course, template_helper=template_helper)
+    $:plugin_manager.call_hook('course_button_hook', template_helper=template_helper, course=course)[0]
```

Comment:
Creating a hook with the same parameters of another hook just below this same hook seems useless.